gapic-generator/lib/gapic/schema/wrappers.rb

```diff
@@ -376,6 +388,10 @@ def initialize descriptor, address, docs, messages, enums, services,
         @services.each { |m| m.parent = self }
       end
 
+      def containing_file
+        self
+      end
```

Comment:
duplication?

Reply:
No, this is intentional. The goal is to return the `Gapic::Schema::File` object in the tree. In the base class, we turn around and ask the parent, but if the node is a File, we override that to return self.